File: historyscan-back/src/main/java/io/sch/historyscan/domain/contexts/codebase/find/CurrentCodeBase.java
package io.sch.historyscan.domain.contexts.codebase.find;

import io.sch.historyscan.domain.hexagonalarchitecture.DDDValueObject;

@DDDValueObject
public record CurrentCodeBase(String name,
                              String url,
                              String currentBranch,
                              boolean gitError) {
}

File: historyscan-back/src/main/kotlin/io/sch/historyscan/domain/contexts/codebase/find/CurrentCodeBase.kt
package io.sch.historyscan.domain.contexts.codebase.find

import io.sch.historyscan.domain.hexagonalarchitecture.DDDValueObject

@DDDValueObject
class CurrentCodeBase(
    val name: String,
    val url: String?,
    val currentBranch: String?,
    val gitError: Boolean = false
)
